### ISSUE TITLE
load wrong icuuc.dll 

### ISSUE THREAD

**Issue by lag945** [pmed/v8-nuget#28]
Win10 has iccuc.dll at system32 , it won't work in many situations because loading system32/iccuc.dll first.
Can this project consider different dll name or version name iccuc.dll like iccuc230.dll?

**Comment by pmed** [comment]
No, I can't. Provided with this package icuuc.dll is created during the whole V8 build process, that can't be changed.

This issue is out of this project scope, please consider other packaging/installation ways dealing with multiple ICU versions.
